static/js/util/date_test.js

```diff
@@ -2,19 +2,31 @@
 import { assert } from 'chai';
 import moment from 'moment';
 
-import { ifValidDate } from './date';
+import { ifValidDate, formatMonthDate } from './date';
 
 describe('ifValidDate', () => {
```

Comment:
we should change this describe name to `'date utility functions'` or something equivalent